docs/devices/E1766.md

```diff
@@ -13,7 +13,7 @@ description: "Integrate your IKEA E1766 via Zigbee2MQTT with whatever smart home
 | Vendor  | IKEA  |
 | Description | TRADFRI open/close remote |
 | Exposes | battery, action, linkquality |
-| Picture | ![IKEA E1766](../images/devices/E1766.jpg) |
+| Picture | ![IKEA E1766](../images/devices/E1766_2.jpg) |
```

Comment:
Please replace the old E1766.jpg instead of adding a new image

Reply:
Alright! I have replaced the old E1766,jpg with the new one! 
Just did not want to destroy somebody elses work ;-)